kerchunk/utils.py

```diff
@@ -116,9 +140,32 @@ def rename_target_files(
         ujson.dump(new, f)
 
 
-def _encode_for_JSON(store):
+def zarr_init_group_and_store(store=None, zarr_version=None, overwrite=True):
```

Comment:
Worth checking whether these new are appropriate for the public API of kerchunk (IMO, no. I'd recommend prefixing the function names with underscores or putting them in a `_utils.py`.